pom.xml

```diff
@@ -329,37 +329,37 @@
       <dependency>
         <groupId>org.apache.commons</groupId>
         <artifactId>commons-lang3</artifactId>
-        <version>3.9</version>
+        <version>3.10</version>
       </dependency>
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient</artifactId>
-        <version>${httpclient.version}</version>
+        <version>4.5.12</version>
```

Comment:
This one still needs to be updated in properties and referenced by `${httpclient.version}`